src/main/java/com/minecolonies/coremod/util/SoundUtils.java

```diff
@@ -123,4 +141,44 @@ public static void playSoundAtCitizenWithChance(@NotNull final World worldIn, @N
                     (float) PITCH);
         }
     }
+
+    /**
+     * Play an interaction sound with chance at the citizen.
+     * @param world the world.
+     * @param position the position.
+     * @param chance the chance.
+     * @param citizen the citizen.
+     */
+    public static void playInteractionSoundAtCitizenWithChance(@NotNull final World world, @NotNull final BlockPos position, final int chance, @NotNull final EntityCitizen citizen)
+    {
+        if (chance > rand.nextInt(ONE_HUNDRED))
+        {
+            String prefix = "";
+
+            if (citizen.getWorkBuilding() != null)
+            {
+                prefix = citizen.getWorkBuilding().getJobName();
+            }
+
+            if("GuardTower".equals(prefix) && citizen.getWorkBuilding() instanceof AbstractBuildingGuards)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'GUARD_TOWER' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1192)